_translator-files/po/de/Running-a-Server.po

```diff
@@ -334,72 +351,81 @@ msgid ""
 "|**Genre Classical/Folk** |`classical.jamulus.io:22524`|\n"
 "|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|\n"
 msgstr ""
+"| Genre |   Server Address           |\n"
+"|-----------|------------------|\n"
+"|**Any Genre 1** |`anygenre1.jamulus.io:22124`|\n"
+"|**Any Genre 2** |`anygenre2.jamulus.io:22224`|\n"
+"|**Any Genre 3** |`anygenre3.jamulus.io:22624`|\n"
+"|**Genre Rock** |`rock.jamulus.io:22424`|\n"
+"|**Genre Jazz** |`jazz.jamulus.io:22324`|\n"
+"|**Genre Classical/Folk** |`classical.jamulus.io:22524`|\n"
+"|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:129
 msgid "You can also specify a [Directory](#3-directory) in the same way from the command line, providing the Server Address in the same format."
-msgstr ""
+msgstr "Du kannst auch ein [Verzeichnis Server](#3-verzeichnis-server) auf dieselbe Weise in der Befehlszeile angeben, indem du die Serveradresse im selben Format angibst."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:131
 msgid "#### Running as a Directory"
-msgstr ""
+msgstr "#### Als Verzeichnis Server betreiben"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:133
 msgid "If you wish to run a [Directory](Running-a-Server#3-directory) please see [this guide](Custom-Directories)."
-msgstr ""
+msgstr "Wenn du einen [Verzeichnis Server](Running-a-Server#3-verzeichnis-server) betreiben möchtest, lese bitte [diese Anleitung](Custom-Directories)."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:135
 msgid "### Maintenance"
-msgstr ""
+msgstr "### Wartung"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:137
 msgid "#### Viewing The Logs"
-msgstr ""
+msgstr "#### Anzeigen der Protokolle - Log Files"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:139
 msgid "Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file."
-msgstr ""
+msgstr "Jamulus protokolliert in der Systemprotokolldatei, wenn du die Einstellung `StandardOutput=journal` in der Unit-Datei nicht verändert hast."
```

Comment:
@gilgongo this is somewhat a contradiction the the no-logging policy in the privacy statement.